_sass/preloader.scss

```diff
@@ -0,0 +1,85 @@
+.loading-screen {
+    background-color: rgba(19, 54, 103, 0.85);
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 9999;
+    animation: fadeOut 1.8s ease-in-out forwards;
```

Comment:
I feel the time could be increased a bit more because the preloader isn't prominently visible. Otherwise I like that you used the MLH theme colours and great job on the code